src/ui/pages/Menu/components/Settings/Settings.types.ts

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from "react";
+
+interface OptionProps {
+  icon: string;
+  label: string;
+  ionIcon?: ReactNode;
```

Comment:
Not sure if this `ionIcon` field is ever passed, if not it can be removed. We have `icon`

Reply:
One thing is the icon in the left, and the the other is the icon on the right.
So far all icons on the right are right arrows, but now we introduce an icon that is a IonToggle that is replacing the previous:
<IonIcon
  aria-hidden="true"
  icon={chevronForward}
  slot="end"
/>
![Screenshot 2024-05-14 at 16 02 59](https://github.com/cardano-foundation/cf-identity-wallet/assets/9588549/cd8ce570-3e2c-40bb-8293-e8c1c06d65bf)

I just renamed it to `actionIcon` to make it more clear it use.